example/server/integrations.py

```diff
@@ -1,20 +1,79 @@
 import logging
 import time
-from typing import List, Dict
+from typing import List, Dict, Optional, Type
 from uuid import uuid4
 
 from django.conf import settings
 from django.db.models import QuerySet
+from django import forms
+
 import polaris.settings
 from polaris.models import Transaction, Asset
-from polaris.integrations import DepositIntegration, WithdrawalIntegration
+from polaris.integrations import (
+    DepositIntegration,
+    WithdrawalIntegration,
+    TransactionForm,
+)
 
 from . import mock_banking_rails as rails
+from .models import PolarisUser, PolarisUserAccount, PolarisUserTransaction
+from .forms import KYCForm
 
 
 logger = logging.getLogger(__name__)
 
 
+def track_user_activity(form: forms.Form, transaction: Transaction):
+    """
+    Creates a PolarisUserTransaction object, and depending on the form
+    passed, also creates a new PolarisUserAccount and potentially a
+    new PolarisUser. This function ensures an accurate record of a
+    particular person's activity.
+    """
+    if isinstance(form, KYCForm):
+        data = form.cleaned_data
+        user = PolarisUser.objects.filter(email=data.get("email")).first()
```

Comment:
Does this mean i can create transactions in anyones account just by putting in their email?

Reply:
Yes.. thats true. What would a normal anchor do? I would guess they'd send a confirmation email and wait to continue processing until the user confirms right?

Reply:
Maybe the best thing is to just not deal with emails, and link every user account to a single stellar account?  I don't really want to design a separate user login system at this point

Reply:
I'm not sure that would really be a useful example. I don't think implementing the confirmation email would be difficult, and the user would only have to do it once. From thereon out, we'll get the PolarisUser from the account the wallet passes us.

This situation also demonstrates that anchor's may want to serve pages without forms during the interactive flow, like when waiting on the user to confirm their email. I filed an issue for this: #78

Reply:
cool i would also be ok with adding a todo and issue so you can land this